lib/intellijCoverageReporter.js

```diff
@@ -33,6 +33,7 @@ function configureAngularCliCoverage(config) {
 function configureCoverage(config) {
   var reporters = config.reporters || [];
   if (cli.isWithCoverage()) {
+    reporters.push('coverage');
```

Comment:
Maybe `reporters.push('coverage')` could be added if `karma-coverage` plugin is used. Looks good to me.
This repo is outdated, let's close this one and continue in the https://youtrack.jetbrains.com/issue/WEB-32836